### PR TITLE
minor text updates on netconf, pyang, pyangbind examples

### PR DESCRIPTION
**Pull request by ksator**

### CODE REVIEW

File: docs/examples/pyangbind/index.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ pip3 freeze | grep pyang
 
 ## Get YANG modules
 
+We need YANG modules
```
Comment (sulrich) [inline docs/examples/pyangbind/index.md:28]: incomplete?

File: docs/examples/pyang/index.md [hunk 2]
```diff
@@ -26,7 +26,9 @@ pip3 freeze | grep pyang
 pip3 freeze | grep pyang
 ```
 
-## We need yang modules
+## Get YANG modules
+
+We need YANG modules
```
Comment (sulrich) [inline docs/examples/pyang/index.md:31]: incomplete?

**Review by sulrich — CHANGES_REQUESTED** [review]
we can address the linting stuff later.

File: docs/examples/pyangbind/index.md (hunk 1 above)
Comment (ksator) [reply docs/examples/pyangbind/index.md:28]: just added `We need YANG modules so we can use Pyang and Pyangbind`

File: docs/examples/pyang/index.md (hunk 2 above)
Comment (ksator) [reply docs/examples/pyang/index.md:31]: just added `We need YANG modules so we can use Pyang`

**Review by sulrich — APPROVED** [review]
good stuff. thanks!

**Pull request merged**